_security-analytics/usage/detectors.md

```diff
@@ -35,6 +35,16 @@ To edit a detector, begin by selecting the link to the detector in the Detector
 After you select the **Alert triggers** tab, you also have the option to add additional alerts for the detector by selecting **Add another alert condition** at the bottom of the page.
 {: .tip }
 
+### Threat intelligence feeds
+
+A threat intelligence feed is a real-time, continuous data stream that gathers information related to risks or threats. The critical information in the tactical threat intelligence feed is called an “indicator of compromise” (IoC). 
+
+As of OpenSearch 2.12, you can enable threat intelligence for Sigma rules related to malicious IP addresses. 
+
+To enable threat intelligence feeds, select the **Enable threat intelligence-based detection** option. 
```

Comment:
I know we can't change this at the moment, but just FYI, it should be "threat-intelligence-based detection".